<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>600</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Projects</string>
  </property>
  <widget class="QPushButton" name="createProject">
   <property name="geometry">
    <rect>
     <x>652</x>
     <y>0</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Project +</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableView">
   <property name="geometry">
    <rect>
     <x>-5</x>
     <y>41</y>
     <width>811</width>
     <height>521</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="homeBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
   <property name="icon">
    <iconset theme="../../static/icons/home.png">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
